opensrp-maternity/src/main/java/org/smartregister/maternity/utils/MaternityUtils.java

```diff
@@ -393,4 +393,28 @@ public static void setActionButtonStatus(Button button, CommonPersonObjectClient
             button.setBackground(ContextCompat.getDrawable(button.getContext(), R.drawable.maternity_outcome_bg));
         }
     }
+
+    @Nullable
+    public static String convertFormNameToType(String formName) {
```

Comment:
check the comments above you dont need this ... its inside the form object.

Reply:
method removed.